### ISSUE TITLE
Shutdown with status 143 after a few minutes

### ISSUE THREAD

**Issue by Blogshot** [iotaledger/iri#412]
My node exits/crashes after a while and shows no usable error messages.

I'm running the 1.4.1.3 IRI with Nelson on a Debian-VM (4 cores, 4 GB RAM, 20 GB HDD).

```
Dez 18 22:54:41 iota systemd[1]: Starting IOTA (IRI) full node...
Dez 18 22:54:41 iota systemd[1]: Started IOTA (IRI) full node.
Dez 18 22:54:42 iota java[15114]: 12/18 22:54:42.242 [main] INFO  com.iota.iri.IRI - Welcome to IRI 1.4.1.3
Dez 18 22:54:45 iota java[15114]: 12/18 22:54:45.767 [main] INFO  c.i.i.s.r.RocksDBPersistenceProvider - Initializing Database Backend...
Dez 18 22:54:45 iota java[15114]: 12/18 22:54:45.836 [main] INFO  c.i.i.s.r.RocksDBPersistenceProvider - RocksDB persistence provider initialized.
Dez 18 22:54:45 iota java[15114]: 12/18 22:54:45.859 [main] INFO  com.iota.iri.network.UDPReceiver - UDP replicator is accepting connections on udp port 14600
Dez 18 22:54:45 iota java[15114]: 12/18 22:54:45.862 [UDP receiving thread] INFO  com.iota.iri.network.UDPReceiver - Spawning Receiver Thread
Dez 18 22:54:45 iota java[15114]: 12/18 22:54:45.862 [main] INFO  c.i.i.network.replicator.Replicator - Started ReplicatorSourcePool
Dez 18 22:54:45 iota java[15114]: 12/18 22:54:45.863 [UDP receiving thread] INFO  com.iota.iri.network.UDPReceiver - Receiver thread processed/dropped ratio: 0/0
Dez 18 22:54:45 iota java[15114]: 12/18 22:54:45.871 [Thread-2] INFO  c.i.i.n.r.ReplicatorSourcePool - TCP replicator is accepting connections on tcp port 15600
Dez 18 22:54:45 iota java[15114]: 12/18 22:54:45.882 [main] INFO  com.iota.iri.network.Node - -> Adding neighbor : /46.101.111.142:14265
Dez 18 22:54:45 iota java[15114]: 12/18 22:54:45.887 [UDP receiving thread] INFO  com.iota.iri.network.UDPReceiver - Receiver thread processed/dropped ratio: 0/0

[...]

Dec 18 23:00:02 iota java[15114]: 12/18 23:00:02.783 [UDP receiving thread] INFO  com.iota.iri.network.UDPReceiver - Receiver thread processed/dropped ratio: 0/0
[ ... 16 repetitions ... ]
Dec 18 23:00:02 iota java[15114]: 12/18 23:00:02.917 [UDP receiving thread] INFO  com.iota.iri.network.UDPReceiver - Receiver thread processed/dropped ratio: 0/0

Dec 18 23:00:02 iota 63: New IRI release 1.4.1.2 detected! Updating installed version 1.4.1.3 to the latest release 1.4.1.2
Dec 18 23:00:02 iota systemd[1]: Stopping IOTA (IRI) full node...
Dec 18 23:00:02 iota java[15114]: 12/18 23:00:02.928 [Shutdown Hook] INFO  com.iota.iri.IRI - Shutting down IOTA node, please hold tight...

Dec 18 23:00:02 iota java[15114]: 12/18 23:00:02.932 [UDP receiving thread] INFO  com.iota.iri.network.UDPReceiver - Receiver thread processed/dropped ratio: 0/0
[ ... 32 repetitions ... ]
Dec 18 23:00:03 iota java[15114]: 12/18 23:00:03.539 [UDP receiving thread] INFO  com.iota.iri.network.UDPReceiver - Receiver thread processed/dropped ratio: 0/0

Dec 18 23:00:03 iota java[15114]: 12/18 23:00:03.921 [pool-2-thread-1] INFO  com.iota.iri.network.Node - Shutting down Broadcaster Thread
Dec 18 23:00:03 iota java[15114]: 12/18 23:00:03.922 [pool-2-thread-4] INFO  com.iota.iri.network.Node - Shutting down Process Received Data Thread
Dec 18 23:00:03 iota java[15114]: 12/18 23:00:03.923 [pool-2-thread-5] INFO  com.iota.iri.network.Node - Shutting down Reply To Request Thread
Dec 18 23:00:04 iota java[15114]: 12/18 23:00:04.132 [pool-2-thread-3] INFO  com.iota.iri.network.Node - Shutting down Neighbor DNS Refresher Thread

Dec 18 23:00:04 iota java[15114]: 12/18 23:00:04.318 [UDP receiving thread] INFO  com.iota.iri.network.UDPReceiver - Receiver thread processed/dropped ratio: 0/0
[ ... 10 repetitions ... ]
Dec 18 23:00:06 iota java[15114]: 12/18 23:00:06.113 [UDP receiving thread] INFO  com.iota.iri.network.UDPReceiver - Receiver thread processed/dropped ratio: 0/0

Dec 18 23:00:06 iota java[15114]: 12/18 23:00:06.128 [pool-2-thread-2] INFO  com.iota.iri.network.Node - Shutting down Requester Thread
Dec 18 23:00:06 iota java[15114]: 12/18 23:00:06.985 [UDP receiving thread] INFO  com.iota.iri.network.UDPReceiver - Shutting down spawning Receiver Thread
Dec 18 23:00:11 iota java[15114]: 12/18 23:00:11.482 [Shutdown Hook] INFO  com.iota.iri.storage.Tangle - Shutting down Tangle Persistence Providers...
Dec 18 23:00:11 iota systemd[1]: iota.service: main process exited, code=exited, status=143/n/a
Dec 18 23:00:11 iota systemd[1]: Stopped IOTA (IRI) full node.
Dec 18 23:00:11 iota systemd[1]: Unit iota.service entered failed state.
```

The node was started at 22:54:42 and exited at 23:00:11. I can't make out an error message besides the exit status 143.

The node is started as systemd service as follows:
```
[Unit]
Description=IOTA (IRI) full node
After=network.target

[Service]
WorkingDirectory=/root/iota
User=root
PrivateDevices=yes
Type=simple
ExecReload=/bin/kill -HUP $MAINPID
KillMode=mixed
KillSignal=SIGTERM
TimeoutStopSec=60
ExecStart=/usr/bin/java -Xmx3G -Djava.net.preferIPv4Stack=true -jar iri-1.4.1.3.jar -c iota.ini
Restart=on-failure
RestartSec=30

[Install]
WantedBy=multi-user.target
Alias=iota.service
```
(Yes I know that running as root is not recommended!)

Configuration file:
```
[IRI]
PORT = 14265
UDP_RECEIVER_PORT = 14600
TCP_RECEIVER_PORT = 15600
API_HOST = 0.0.0.0
IXI_DIR = ixi
HEADLESS = true
DEBUG = false
TESTNET = false
DB_PATH = mainnetdb
RESCAN_DB = false
REMOTE = true

REMOTE_LIMIT_API="interruptAttachingToTangle, attachToTangle, getNeighbors"
# removeNeighbors, addNeighbors allowed because of Nelson

NEIGHBORS = *censored*
```

I also noticed:
- IRI release 1.4.1.2 is detected and installed? (`Updating installed version 1.4.1.3  to the latest release 1.4.1.2`)
- The processed/dropped ratio is always 0/0 for the short time the node is running.
- This also happened before installing Nelson,

What can be done here? 

Update: 1.4.1.4 was released two hours after this report, I will test it shortly. (Even if the changelog doesn't address my issue)

**Comment by Blogshot** [comment]
I just noticed that the release I'm using is a testnet-release. I will try the newly released 1.4.1.4 for the mainnet when I'm home and will test accordingly.

**Comment by Blogshot** [comment]
After testing the mainnet executable I found that the node is stable and processes transactions as planned.
